recipes/julia/meta.yaml

```diff
@@ -0,0 +1,51 @@
+package:
+  name: julia
+  version: "0.5.2"
+
+source:
+  fn: v0.5.2.tar.gz
+  url: https://github.com/JuliaLang/julia/archive/v0.5.2.tar.gz
```

Comment:
Would template out the version as in the [`example`]( https://github.com/conda-forge/staged-recipes/blob/31bfcc36b5ab4021ddfb326bd62f5f1cd98af572/recipes/example/meta.yaml ). Should make it easier to update.